packages/angular/cli/models/command-runner.ts

```diff
@@ -17,26 +17,41 @@ import {
 } from '@angular-devkit/core';
 import * as debug from 'debug';
 import { readFileSync } from 'fs';
-import { dirname, join, resolve } from 'path';
-import { findUp } from '../utilities/find-up';
+import { join, resolve } from 'path';
 import { parseJsonSchemaToCommandDescription } from '../utilities/json-schema';
 import { UniversalAnalytics, getGlobalAnalytics, getSharedAnalytics } from './analytics';
 import { Command } from './command';
-import {
-  CommandDescription,
-  CommandDescriptionMap,
-  CommandWorkspace,
-} from './interface';
+import { CommandDescription, CommandWorkspace } from './interface';
 import * as parser from './parser';
 
 const analyticsDebug = debug('ng:analytics:commands');
 
+const standardCommands = {
```

Comment:
You not simply read this from the `commands.json`, rather than having it duplicated here.

Reply:
The data is constant so there’s really no reason to find, read, and parse a json file on every execution.

Reply:
The file itself can not be removed yet as it is unfortunately used by one of the validation tests due to the top level help command not provided a json help option (unlike the command specific help).  Once this is remedied, the file can be removed as well.

Reply:
Having this duplicated means at some point we'll change one and forget to change the other. If we can't remove it now then we should have a warning here.

Reply:
Done.